content/nic/installation/upgrade-version.md

```diff
@@ -0,0 +1,311 @@
+---
+# We use sentence case and present imperative tone
+title: "Upgrade NGINX Ingress Controller"
+# Weights are assigned in increments of 100: determines sorting order
+weight: 500
+# Creates a table of contents and sidebar, useful for large documents
+toc: true
+# Types have a 1:1 relationship with Hugo archetypes, so you shouldn't need to change this
+nd-content-type: how-to
+# Intended for internal catalogue and search, case sensitive:
+# Agent, N4Azure, NIC, NIM, NGF, NAP-DOS, NAP-WAF, NGINX One, NGINX+, Solutions, Unit
+nd-product: NIC 
+---
+
+This document describes how to upgrade F5 NGINX Ingress Controller when a new version releases. 
+
+It covers the necessary steps for minor versions as well as major versions (Such as 3.x to 4.x).
+
+Many of the nuances in upgrade paths relate to how custom resource definitions (CRDs) are managed.
+
+## Minor NGINX Ingress Controller upgrades
+
+### Upgrade NGINX Ingress Controller CRDs
+
+{{< call-out "note" >}} If you are running NGINX Ingress Controller v3.x, you should read [Upgrade from NGINX Ingress Controller v3.x to v4.0.0]({{< ref "/nic/installation/upgrade-version.md#upgrade-from-3x-to-4x" >}}) before continuing. {{< /call-out >}}
+
+To upgrade the CRDs, pull the Helm chart source, then use _kubectl apply_:
+
+```shell
+helm pull oci://ghcr.io/nginx/charts/nginx-ingress --untar --version {{< nic-helm-version >}}
+kubectl apply -f crds/
+```
+
+Alternatively, CRDs can be upgraded without pulling the chart by running:
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/nginx/kubernetes-ingress/v{{< nic-version >}}/deploy/crds.yaml
+```
+
+In the above command, `v{{< nic-version >}}` represents the version of NGINX Ingress Controller release rather than the Helm chart version.
```

Comment:
```suggestion
In the above command, `v{{< nic-version >}}` represents the version of the NGINX Ingress Controller release rather than the Helm chart version.
```